test/DurableTask.SqlServer.AzureFunctions.Tests/IntegrationTestBase.cs

```diff
@@ -21,21 +23,31 @@ namespace DurableTask.SqlServer.AzureFunctions.Tests
 
     public class IntegrationTestBase : IAsyncLifetime
     {
+        private readonly string taskHubName;
```

Comment:
Please remove `private` for consistency with the rest of the code. This code base doesn't use default visibility modifiers.